openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1916,6 +1916,27 @@ The following is a non-normative example of a Credential Request with Credential
 
 The value of the `credential` claim in the Credential Response MUST be a string that is the base64url-encoded representation of the issued Credential.
 
+### IACA Certificate Metadata
+
+This profile defines an OpenID4VCI Credential Issuer Metadata parameter to publish Issuing Authority Certificate Authority (IACA) certificates.  These certificates are defined in ISO 18013-5 (mDL).  They are needed to verify mdoc certificate chains.  This parameter is:
+
```

Comment:
If I got it well, I recap:

- mdoc_iacas_uri points to an HTTPs resource that publishes a json object containing trust anchors' certificates needed to verify the mdoc issued by a VCI

may we use the certificate's CN/Subject as key name, instead of `"certificate"`? that would help the resolution of the issuer in a human readable way.

or, wdyt about having the certificate issuer's entity id in addition to that "certificate" member name?